packages/venia-styleguide/webpack.config.js

```diff
@@ -37,8 +37,9 @@ module.exports = () => ({
                     }
                 ]
             },
+
             {
-                test: /\.md$/,
+                test: /\.md$|\.mdx$/,
```

Comment:
This was intentional; just as we avoid the `.jsx` extension, I'd like to avoid the `.mdx` extension here in favor of just `.md`.

Reply:
Does this mean, no more `mdx` files? I have a bunch. If so, I can update them all to `md`.

Reply:
Done

Reply:
Correct. `.md` is a valid extension for MDX, and it's the one I chose to favor here.